Library/Homebrew/formula_auditor.rb

```diff
@@ -339,6 +339,17 @@ def audit_postgresql
       end
     end
 
+    def audit_glibc
+      return unless formula.name == "glibc"
+      return unless @core_tap
+
+      version = formula.version.to_s
+      return if version == "2.23"
+
+      problem "The glibc version can only be 2.23 as this is the version used by our CI on Linux. " \
+              "Do not update glibc else this might break bottles for users that have brewed glibc installed."
```

Comment:
Is that the reason? I thought newer `glibc` was ABI backwards compatible.

Reply:
I changed the text, actually it should not break anything for most of our users; but for those that have an older glibc version, these just need the same version as the one we use on CI.